lib/graphQl/joiConverter.js

```diff
@@ -45,7 +44,8 @@ joiConverter.swap = (joiScheme, graphQlResources) => {
   if (!joiScheme._settings) {
     type = joiConverter.simpleAttribute(joiScheme)
   } else {
-    const otherType = joiScheme._settings.__one || joiScheme._settings.__many
+    let otherType = joiScheme._settings.__one || joiScheme._settings.__many
+    otherType = otherType.join('_PluS_')
```

Comment:
I would put `_PluS_` in a constant and reuse that, in case the separator needs to be renamed (it's highly unlikely, but someone might include `_PluS_` in their resource names.